scrapers/StraponDreamer.yml

```
@@ -0,0 +1,21 @@
+name: StraponDreamer
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - strapondreamer.com/videos
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@id="videoWrapper"]/div[4]/div[2]
```

We generally avoid using indexes if possible, for this case isnt a simple `//h1` selector enough?
```suggestion
      Title: //h1
```

---

You're right, much simpler.